services/dy-jupyter-extensions/neuron/docker/boot_notebook.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# overrides the base script
+
+set -e
+
+# try to pull data from S3
+python /docker/state_puller.py ${SIMCORE_NODE_APP_STATE_PATH} --silent
+
+# the notebooks in the folder shall be trusted by default
+#jupyter trust ${SIMCORE_NODE_APP_STATE_PATH}/*
+
+# Trust all notebooks in the notbooks folder
+find ${SIMCORE_NODE_APP_STATE_PATH} -name '*.ipynb' | xargs jupyter trust
+
+# call the notebook with the basic parameters
+start-notebook.sh \
+    --NotebookApp.base_url=${SIMCORE_NODE_BASEPATH} \
+    --NotebookApp.extra_static_paths="['${SIMCORE_NODE_BASEPATH}/static']" \
+    --NotebookApp.notebook_dir=${SIMCORE_NODE_APP_STATE_PATH} \
+    --NotebookApp.token=""  \
+    --NotebookApp.disable_check_xsrf='True' \
+    --NotebookApp.quit_button='False' \
+    --NotebookApp.webbrowser_open_new='0' \
+    --NotebookApp.nbserver_extensions="{'input_retriever':True, 'state_handler':True}" \
```

Comment:
why not move this file to the base notebook script?

Reply:
Th only change is that I trust all the notebooks in the folder, does this also work for the others? Then i will just change the corresponding lins 11 and 14

Reply:
the idea is to have consistency in all the notebook-based services...

Reply:
All in the base now

Reply:
ok I thought this was mostly the same as what is in the base boot_notebook.sh file.